scripts/zones/Ordelles_Caves/npcs/qm6.lua

```diff
@@ -6,32 +6,25 @@
 -----------------------------------
 package.loaded["scripts/zones/Ordelles_Caves/TextIDs"] = nil;
 -----------------------------------
-require("scripts/globals/settings");
 require("scripts/zones/Ordelles_Caves/TextIDs");
+require("scripts/globals/npc_util");
 -----------------------------------
 
 function onTrade(player,npc,trade)
-
-    if (player:getVar("darkPuppetCS") >= 4) then
-        if (trade:hasItemQty(16940,1) and trade:getItemCount() == 1) then -- Trade Gerwitz's Sword
-            player:tradeComplete();
-            player:messageSpecial(GERWITZS_SOUL_DIALOG);
-            SpawnMob(17568137):updateClaim(player);
-        end
+    -- DARK PUPPET: Gerwitz's Sword
+    if ( player:getVar("darkPuppetCS") >= 4 and npcUtil.tradeHas(trade, 16940) ) then
```

Comment:
and here

Reply:
I've kept up in the `if (stuff) then` simply because thats what the entire project has always done but some of the more recent scripts by demo and kj have started to go to `if stuff then` so maybe its a good time to get in irc and have a good discussion about style guide additions+general cleanup of old code

@takhlaq @teschnei @wrenffxi

Reply:
i'll probably mass replace them at some point to remove extra parens once bcnms and quests are done